<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Grafik</class>
 <widget class="QDialog" name="Grafik">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1" rowspan="11">
    <widget class="QGraphicsView" name="graphicsView"/>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string>Q2 = f(t)</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>Промежутки</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>hi = f(t)</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="pushButton_7">
     <property name="text">
      <string>sigma = f(t)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Q1 = f(t)</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButton_4">
     <property name="text">
      <string>Q = f(t)</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="pushButton_5">
     <property name="text">
      <string>mu1 = f(t)</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="pushButton_6">
     <property name="text">
      <string>mu2 = f(t)</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QPushButton" name="pushButton_8">
     <property name="toolTip">
      <string>Сохранить в формате данных, разделённых точкой с запятой для обработки в Excel</string>
     </property>
     <property name="text">
      <string>Сохранить csv</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QPushButton" name="pushButton_9">
     <property name="toolTip">
      <string>Сохранить результаты расчёта в формате для чтения программой Waves (*.wdt)</string>
     </property>
     <property name="text">
      <string>Импорт для Waves</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
